diff --git a/node_modules/expo-file-system/.eslintrc.js b/node_modules/expo-file-system/.eslintrc.js
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/CHANGELOG.md b/node_modules/expo-file-system/CHANGELOG.md
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/README.md b/node_modules/expo-file-system/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/android/build.gradle b/node_modules/expo-file-system/android/build.gradle
old mode 100644
new mode 100755
index 4a6df85..9146be0
--- a/node_modules/expo-file-system/android/build.gradle
+++ b/node_modules/expo-file-system/android/build.gradle
@@ -54,12 +54,12 @@ android {
   compileSdkVersion safeExtGet("compileSdkVersion", 33)
 
   compileOptions {
-    sourceCompatibility JavaVersion.VERSION_11
-    targetCompatibility JavaVersion.VERSION_11
+    sourceCompatibility JavaVersion.VERSION_17
+    targetCompatibility JavaVersion.VERSION_17
   }
 
   kotlinOptions {
-    jvmTarget = JavaVersion.VERSION_11.majorVersion
+    jvmTarget = "17"
   }
 
   namespace "expo.modules.filesystem"
diff --git a/node_modules/expo-file-system/android/src/main/AndroidManifest.xml b/node_modules/expo-file-system/android/src/main/AndroidManifest.xml
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/AppDirectoriesModule.kt b/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/AppDirectoriesModule.kt
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/CountingRequestBody.kt b/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/CountingRequestBody.kt
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FilePermissionModule.kt b/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FilePermissionModule.kt
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemExceptions.kt b/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemExceptions.kt
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemFileProvider.kt b/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemFileProvider.kt
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.kt b/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.kt
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemPackage.kt b/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemPackage.kt
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemRecords.kt b/node_modules/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemRecords.kt
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/android/src/main/res/xml/file_system_provider_paths.xml b/node_modules/expo-file-system/android/src/main/res/xml/file_system_provider_paths.xml
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/app.plugin.js b/node_modules/expo-file-system/app.plugin.js
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/ExponentFileSystem.d.ts b/node_modules/expo-file-system/build/ExponentFileSystem.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/ExponentFileSystem.d.ts.map b/node_modules/expo-file-system/build/ExponentFileSystem.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/ExponentFileSystem.js b/node_modules/expo-file-system/build/ExponentFileSystem.js
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/ExponentFileSystem.js.map b/node_modules/expo-file-system/build/ExponentFileSystem.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/ExponentFileSystem.web.d.ts b/node_modules/expo-file-system/build/ExponentFileSystem.web.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/ExponentFileSystem.web.d.ts.map b/node_modules/expo-file-system/build/ExponentFileSystem.web.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/ExponentFileSystem.web.js b/node_modules/expo-file-system/build/ExponentFileSystem.web.js
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/ExponentFileSystem.web.js.map b/node_modules/expo-file-system/build/ExponentFileSystem.web.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/ExponentFileSystemShim.d.ts b/node_modules/expo-file-system/build/ExponentFileSystemShim.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/ExponentFileSystemShim.d.ts.map b/node_modules/expo-file-system/build/ExponentFileSystemShim.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/ExponentFileSystemShim.js b/node_modules/expo-file-system/build/ExponentFileSystemShim.js
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/ExponentFileSystemShim.js.map b/node_modules/expo-file-system/build/ExponentFileSystemShim.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/FileSystem.d.ts b/node_modules/expo-file-system/build/FileSystem.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/FileSystem.d.ts.map b/node_modules/expo-file-system/build/FileSystem.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/FileSystem.js b/node_modules/expo-file-system/build/FileSystem.js
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/FileSystem.js.map b/node_modules/expo-file-system/build/FileSystem.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/FileSystem.types.d.ts b/node_modules/expo-file-system/build/FileSystem.types.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/FileSystem.types.d.ts.map b/node_modules/expo-file-system/build/FileSystem.types.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/FileSystem.types.js b/node_modules/expo-file-system/build/FileSystem.types.js
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/FileSystem.types.js.map b/node_modules/expo-file-system/build/FileSystem.types.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/index.d.ts b/node_modules/expo-file-system/build/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/index.d.ts.map b/node_modules/expo-file-system/build/index.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/index.js b/node_modules/expo-file-system/build/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/index.js.map b/node_modules/expo-file-system/build/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/types.d.ts b/node_modules/expo-file-system/build/types.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/types.d.ts.map b/node_modules/expo-file-system/build/types.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/types.js b/node_modules/expo-file-system/build/types.js
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/build/types.js.map b/node_modules/expo-file-system/build/types.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/expo-module.config.json b/node_modules/expo-file-system/expo-module.config.json
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem.podspec b/node_modules/expo-file-system/ios/EXFileSystem.podspec
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXFilePermissionModule.h b/node_modules/expo-file-system/ios/EXFileSystem/EXFilePermissionModule.h
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXFilePermissionModule.m b/node_modules/expo-file-system/ios/EXFileSystem/EXFilePermissionModule.m
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystem.h b/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystem.h
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystem.m b/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystem.m
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystemAssetLibraryHandler.h b/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystemAssetLibraryHandler.h
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystemAssetLibraryHandler.m b/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystemAssetLibraryHandler.m
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystemLocalFileHandler.h b/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystemLocalFileHandler.h
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystemLocalFileHandler.m b/node_modules/expo-file-system/ios/EXFileSystem/EXFileSystemLocalFileHandler.m
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionCancelableUploadTaskDelegate.h b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionCancelableUploadTaskDelegate.h
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionCancelableUploadTaskDelegate.m b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionCancelableUploadTaskDelegate.m
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionDownloadTaskDelegate.h b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionDownloadTaskDelegate.h
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionDownloadTaskDelegate.m b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionDownloadTaskDelegate.m
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionHandler.h b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionHandler.h
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionHandler.m b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionHandler.m
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionResumableDownloadTaskDelegate.h b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionResumableDownloadTaskDelegate.h
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionResumableDownloadTaskDelegate.m b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionResumableDownloadTaskDelegate.m
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionTaskDelegate.h b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionTaskDelegate.h
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionTaskDelegate.m b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionTaskDelegate.m
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionTaskDispatcher.h b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionTaskDispatcher.h
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionTaskDispatcher.m b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionTaskDispatcher.m
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionUploadTaskDelegate.h b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionUploadTaskDelegate.h
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionUploadTaskDelegate.m b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXSessionUploadTaskDelegate.m
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXTaskHandlersManager.h b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXTaskHandlersManager.h
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXTaskHandlersManager.m b/node_modules/expo-file-system/ios/EXFileSystem/EXSessionTasks/EXTaskHandlersManager.m
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/NSData+EXFileSystem.h b/node_modules/expo-file-system/ios/EXFileSystem/NSData+EXFileSystem.h
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/EXFileSystem/NSData+EXFileSystem.m b/node_modules/expo-file-system/ios/EXFileSystem/NSData+EXFileSystem.m
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/ios/Tests/EXFileSystemSpec.swift b/node_modules/expo-file-system/ios/Tests/EXFileSystemSpec.swift
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/plugin/build/withFileSystem.d.ts b/node_modules/expo-file-system/plugin/build/withFileSystem.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/plugin/build/withFileSystem.js b/node_modules/expo-file-system/plugin/build/withFileSystem.js
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/plugin/src/withFileSystem.ts b/node_modules/expo-file-system/plugin/src/withFileSystem.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/plugin/tsconfig.json b/node_modules/expo-file-system/plugin/tsconfig.json
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/src/ExponentFileSystem.ts b/node_modules/expo-file-system/src/ExponentFileSystem.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/src/ExponentFileSystem.web.ts b/node_modules/expo-file-system/src/ExponentFileSystem.web.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/src/ExponentFileSystemShim.ts b/node_modules/expo-file-system/src/ExponentFileSystemShim.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/src/FileSystem.ts b/node_modules/expo-file-system/src/FileSystem.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/src/FileSystem.types.ts b/node_modules/expo-file-system/src/FileSystem.types.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/src/index.ts b/node_modules/expo-file-system/src/index.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/src/types.ts b/node_modules/expo-file-system/src/types.ts
old mode 100644
new mode 100755
diff --git a/node_modules/expo-file-system/tsconfig.json b/node_modules/expo-file-system/tsconfig.json
old mode 100644
new mode 100755
